### PR TITLE
Add Japanese translations

### PR DESCRIPTION
**Pull request by yuh-pen**
This adds the Japanese translations for the error messages 🧡 
Best regards

## Pre-Merge Checklist
- [x] CHANGELOG.md updated with short summary

### CODE REVIEW

**Comment by grosser** [comment]
5.12.0

File: rails/locales/ja.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+en:
```
Comment (serggl) [inline rails/locales/ja.yml:1]: @grosser there is an error in this PR so it overrides default english translations with japanese texts

Comment (yujideveloper) [reply rails/locales/ja.yml:1]: I created PR #419

Comment (grosser) [reply rails/locales/ja.yml:1]: 5.12.1

Comment (yuh-pen) [reply rails/locales/ja.yml:1]: Sorry; Thank you.

**Pull request merged**